LICENSE.md

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is supposed to be called `LICENSE` and not `LICENSE.md`, that's why the link in the `README.md` is broken. See below 👇🏽 

![Screenshot 2023-02-15 at 22 04 46](https://user-images.githubusercontent.com/45739963/219257712-6c1e5689-ab40-4569-9363-f6a07b6a2f0b.png)